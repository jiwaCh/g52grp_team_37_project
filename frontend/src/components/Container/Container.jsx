import React from 'react';
import PropTypes from 'prop-types';
import MUContainer from '@material-ui/core/Container';
import useStyles from './Container.styles';

const Container = ({ children }) => {
  const classes = useStyles();

  return <MUContainer className={classes.root}>{children}</MUContainer>;
};

Container.propTypes = {
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
};

export default Container;
