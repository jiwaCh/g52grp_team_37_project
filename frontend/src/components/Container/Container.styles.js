import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  root: {
    marginTop: theme.spacing(1),
    marginBottom: theme.spacing(1),
  },
}));

export default useStyles;
