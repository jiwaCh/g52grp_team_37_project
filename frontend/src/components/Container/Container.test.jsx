import React from 'react';
import Container from './Container';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

it('renders correctly', () => {
  const component = shallow(<Container />);
  expect(toJson(component)).toMatchSnapshot();
});
