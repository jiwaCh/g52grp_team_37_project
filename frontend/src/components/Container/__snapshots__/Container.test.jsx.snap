// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-root-1"
/>
`;
